hw/ip/usbdev/data/usbdev_testplan.hjson

```diff
@@ -431,8 +431,8 @@
             then write will only have an effect when the device is in the LinkPowered state.
 
             - Perform no operation in J state for more than 3 ms.
-            - Read the link state from usbstate register to see if it is in powered state and then send the
-              resume signal (K state) for 20 ms and set the usbctrl.resume_link_active.
+            - Read the link state from usbstate register to see if it is in powered state and set the
+              usbctrl.resume_link_active.
             - Verify that the resume_link_active interrupt goes high and device will be resumed.
```

Comment:
The description change is okay but the register name is 'usbstat' without a trailing 'e.'

Also, whilst this test description is being changed, please note that there is no 'resume_link_active interrupt' and the test should instead be looking at the link state in 'usbstat' to ascertain whether the 'resuming' state has been entered.

Reply:
> The description change is okay but the register name is 'usbstat' without a trailing 'e.'
> 
> Also, whilst this test description is being changed, please note that there is no 'resume_link_active interrupt' and the test should instead be looking at the link state in 'usbstat' to ascertain whether the 'resuming' state has been entered.

Are you suggesting that the "link_resume" interrupt in the intr_state register doesn't monitor the resuming state?